### PR TITLE
FZ83P-6

### PR DESCRIPTION
**Pull request by bartzaw**
I've added flex and rebuilt features section to be fully mobile-first. I removed some, in my opinion, unnecessary tags and classes. Created separate div for headers to better position different parts of section. Moreover, added flex to sections: features, add-info & form to see how they react.

### CODE REVIEW

File: assets/style/style.css [hunk 1]
```diff
@@ -69,6 +69,18 @@ h2 {
     transition: 1s;
 }
 
+.main-container {
+    display: flex;
+    flex-wrap: nowrap;
```
Comment (sebastian-kreft) [inline assets/style/style.css]: this is default value, not needed here

File: assets/style/style.css [hunk 2]
```diff
@@ -160,6 +172,8 @@ h2 {
 
 .section-form {
     padding-bottom: 15px;
+    display: flex;
+    flex: 1;
```
Comment (sebastian-kreft) [inline assets/style/style.css:175]: Are all elements with this class both flex-container and flex-item? `flex` declaration works only on flex-item

File: assets/style/style.css [hunk 3]
```diff
@@ -204,74 +218,55 @@ h2 {
...
+    text-align: center;
 }
 
-.features-item-header {
-    padding-top: 10px;
-    padding-bottom: 10px;
+.section-add-info {
+    /*display: flex;*/
+    /*flex-grow:1;*/
+    /*flex-direction: column;*/
```
Comment (sebastian-kreft) [inline assets/style/style.css]: Do not leave comments in the code. If you want to have it for later, you can stash this change (as you learned on last lesson).

**Review by sebastian-kreft — COMMENTED** [review]
I left some comments regarding css

**Comment by Decomplosed** [comment]
Yeah, minor changes according to Sebastian's comments, rest is fine for me

**Review by icelandico — APPROVED** [review]
Only perform changes suggested by @sebastian-kreft

File: assets/style/style.css (hunk 1 above)
Comment (bartzaw) [reply assets/style/style.css]: thanks, removed

File: assets/style/style.css (hunk 2 above)
Comment (bartzaw) [reply assets/style/style.css:175]: only sections - I wanted them to act flexible inside of container (class of 'main') and elements inside of section to act the same way.

File: assets/style/style.css (hunk 3 above)
Comment (bartzaw) [reply assets/style/style.css]: ah, yeah. Thanks for pointing that one out, I simply forgot to remove it before pull request, it's gone now.

**Comment by bartzaw** [comment]
@Decomplosed @sebastian-kreft changes made, please check if code is ok to merge

**Pull request merged**